serverless.yml

```diff
@@ -1,5 +1,5 @@
 service: zorgbort
-frameworkVersion: '2'
+frameworkVersion: '3'
 useDotenv: true
```

Comment:
I vaguely recall some kind of warning about this line at some point. Maybe dotenv was integrated in v3 or removed... something... If you have some time to investigate this I'd like to know before merging. If not I'll dig into it when I have a spare moment.

Reply:
The info you seek is in https://www.serverless.com/framework/docs/guides/upgrading-v3:

> Unlike planned initially, loading .env files is kept opt-in via useDotenv: true.

It's under "Deprecated features that will be kept in v3".

Reply:
Alrighty! Thanks for checking and for a good reason why my brain and the deprecation warnings in CI weren't in alignment.